### PR TITLE
worker: Simplify use of job_settings hash lookup

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: lib/OpenQA/Worker/Engines/isotovideo.pm [hunk 1]
```diff
@@ -273,52 +262,16 @@ sub do_asset_caching {
...
-        OPENQA_URL      => $openqa_url,
+    my %vars = (
+        OPENQA_URL      => $webui_host,
         WORKER_INSTANCE => $instance,
-        WORKER_ID       => $workerid,
+        WORKER_ID       => $worker_id,
         PRJDIR          => OpenQA::Utils::sharedir(),
-        %$job_settings
-    );
+        %$job->info->{settings});
```
Comment (okurz) [inline lib/OpenQA/Worker/Engines/isotovideo.pm:274]: this triggers `Can't locate object method "info" via package "21"`. Anyone has an idea how to solve that better?

Comment (perlpunk) [reply lib/OpenQA/Worker/Engines/isotovideo.pm:274]:
```suggestion
        %{ $job->info->{settings} })
```

Comment (okurz) [reply lib/OpenQA/Worker/Engines/isotovideo.pm:274]: Right, that gives me `Use of uninitialized value in concatenation (.) or string`

Comment (perlpunk) [reply lib/OpenQA/Worker/Engines/isotovideo.pm:274]: there is no string concatenation happening in this part of the code?

Comment (Martchus) [reply lib/OpenQA/Worker/Engines/isotovideo.pm:274]: I was wondering about the same. Generally the suggested code change is supposed to work.

Comment (okurz) [reply lib/OpenQA/Worker/Engines/isotovideo.pm:274]: ah ok, I see that it's now about a different line. Well, that helps, I can go on :)

Comment (okurz) [reply lib/OpenQA/Worker/Engines/isotovideo.pm]: Well, couldn't understand that part while being in another casual video meeting so for now I just left out that commit :)

**Comment by Martchus** [comment]
Looks generally good (besides the missing patch coverage).

**Review by Martchus — APPROVED** [review]

**Comment by perlpunk** [comment]
checkstyle failed

**Comment by perlpunk** [comment]
https://github.com/os-autoinst/openQA/pull/4078 for a fix

**Comment by okurz** [comment]
thanks. I wonder if maybe auto-merge skips some checks and maybe that was my problem?

**Pull request merged**